### PR TITLE
Initial Multi-Tenant Support

### PR DESCRIPTION
**Pull request by jasondegraw**
This is a first attempt at better support for describing multi-tenant situations in the schema. I've also added a completely new example file to avoid further dilution of the example files we already have. The example file does not presently include sufficient ownership information, which will be needed before this get merged. Questions that'll need to be addressed include:

- I added a tenant ID element in places where there was a primary contact ID except at the site level. Is that sufficient?
- Is there sufficient support for ownership information?

### CODE REVIEW

**Comment by jasondegraw** [comment]
@macumber @nllong Would either of you have time to review this PR for adding tenant info? We could also ask for working group input. Thoughts on that?

File: BuildingSync.xsd [hunk 1]
```diff
@@ -2200,7 +2210,7 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:element ref="auc:Address" minOccurs="0"/>
-			<xs:element name="ContactTelephoneNumbers" minOccurs="0" maxOccurs="unbounded">
+			<xs:element name="ContactTelephoneNumber" minOccurs="0" maxOccurs="unbounded">
```
Comment (nllong) [inline BuildingSync.xsd]: nice catch.

File: examples/MultitenantBySubsections.xml [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+          </Facility>
+        </Facilities>
+      </Site>
+    </Sites>
+    <Contacts>
+      <Contact ID="Contact1">
+        <ContactRole>Occupant</ContactRole>
+        <ContactName>Arthur Dent</ContactName>
+        <ContactCompany>Colorado Phone Sanitizers Association</ContactCompany>
+        <ContactTelephoneNumber>
```
Comment (nllong) [inline examples/MultitenantBySubsections.xml]: I do think that we need a ContactTelephoneNumbers surrounding this element. So disregard my previous comment.

**Comment by jasondegraw** [comment]
@nllong I've added the container objects you asked for, in both locations where we had telephone number and email addresses.

**Pull request merged**